zap/src/main/java/org/zaproxy/zap/extension/anticsrf/ExtensionAntiCSRF.java

```diff
@@ -492,7 +492,26 @@ public String generateForm(int hrefId) throws Exception {
      * @since 2.7.0
      */
     public String generateForm(HttpMessage msg) throws UnsupportedEncodingException {
+        return generateForm(msg, new HashMap<String, String>());
+    }
+
+    /**
+     * Generates a HTML form from the given message.
+     *
+     * @param msg the message used to generate the HTML form, must not be {@code null}.
+     * @param optionalParams optional parameters for form
+     * @return a string containing the HTML form, never {@code null}.
+     * @throws UnsupportedEncodingException if an error occurred while encoding the values of the
+     *     form.
+     * @since 2.7.0
```

Comment:
?

Reply:
Next version is currently slated to be 2.12.0

Reply:
Better not make it part of the public API.

Reply:
> Better not make it part of the public API.

How can I make it not part of public API?

Reply:
Just remove the `public` keyword.

Reply:
yeah i tried to remove it or make it private, but than i am getting 
`Detected binary changes between zap-2.12.0-SNAPSHOT.jar and zap-2.11.0.jar.
`

Reply:
You might have changed the existing method instead, the report should have more details.

Reply:
The since should be removed.